docs/content/tutorial/ingress-tutorial.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Exposing functions with Ingress"
+draft: false
+weight: 43
+---
+
+This tutorial will walk you through exposing a function using a ingress controller. We will make the function available on a FQDN using Fission's route and ingress controller setup in a cloud environment.
```

Comment:
Okay maybe I am wrong about this but, I think it should be "an ingress"

Reply:
Maybe add a link to a generic explanation of what an ingress controller is. Maybe some page in the kubernetes guide?

Reply:
Unroll the FQDN acronym here. I needed to look it up what it exactly meant.